apps/rush-lib/src/cli/logic/pnpm/PnpmShrinkwrapFile.ts

```diff
@@ -124,9 +135,13 @@ export class PnpmShrinkwrapFile extends BaseShrinkwrapFile {
 
   /**
    * abstract
```

Comment:
>abstract [](start = 5, length = 8)

"abstract" is a TypeScript keyword.  In the base class, could you use comments to annotate which functions are meant to be "virtual", and in the child class could you annotate which functions are "override"ing a base implementation?

I had trouble figuring that out when reading the code.